parquet-column/src/main/java/org/apache/parquet/column/statistics/Statistics.java

```diff
@@ -67,6 +71,27 @@ public static Statistics getStatsBasedOnType(PrimitiveTypeName type) {
     }
   }
 
+  public static Statistics<?> getStatsBasedOnType(PrimitiveTypeName type, OriginalType logicalType) {
```

Comment:
If we are still using `Statistics` classes based on the primitive type, then why was support for stats other than `BinaryStatistics` removed from parquet-cli?

Reply:
Sorry, but I don't get it. Where was the support for stats other than `BinaryStatistics` removed?